homeassistant/components/google_drive/const.py

```diff
@@ -0,0 +1,10 @@
+"""Constants for google drive integration."""
+from __future__ import annotations
+
+from typing import Final
+
+DOMAIN = "google_drive"
+
+DATA_CONFIG_ENTRY: Final = "config_entry"
+DEFAULT_NAME = "Google Drive"
+DEFAULT_ACCESS = "https://www.googleapis.com/auth/drive.file"
```

Comment:
I really like this first simple implementation where the integration only has access to a new google sheet that it created.  That is great from a security perspective.

I imagine for your use case you you told me about, you need to open this up to give access to other specific sheets? I am curious if you have thought about what the config flow would look like for this (e.g. config flow can either  create a new sheet, open an existing sheet w/ read/write access, etc) or if there are other principles here you are thinking about?

Reply:
The point is to keep this PR small. I plan on adding support for access to all spreadsheets from that account. I could not gather from the Google docs how to grant access to a specific file other than just creating one with the app itself. The current scope would be the default behavior. I imagine being able to flip the switch in config options like you have it in calendar.

Reply:
Yes, i'm not suggesting you add this now in this PR, i agree with keeping it small.

Was just trying to get a feel for where you thought this might go in case it comes up in the future (people will ask for it so wanted to be prepared)